graylog2-server/src/main/java/org/graylog2/migrations/MigrationsModule.java

```diff
@@ -24,6 +24,12 @@
 public class MigrationsModule extends PluginModule {
     @Override
     protected void configure() {
+
+        // Make sure there is always a binder for migration modules
```

Comment:
Let's remove the first line as suggested in my diff yesterday. I think it's too generic. Also, please add the second comment from my suggestion to the migration class. Thanks!

```diff
--- graylog2-server/src/main/java/org/graylog2/migrations/MigrationsModule.java
+++ graylog2-server/src/main/java/org/graylog2/migrations/MigrationsModule.java
@@ -52,7 +52,8 @@ public class MigrationsModule extends PluginModule {
         addMigration(V20201103145400_LegacyAuthServiceMigration.class);
         addMigration(V20211221144300_GeoIpResolverConfigMigration.class);
 
-        // Make sure there is always a binder for migration modules
+        // The LegacyAuthServiceMigration is pluggable. This call ensures that there is always a binder present
+        // even if there are no plugins for this migration registered.
         Multibinder.newSetBinder(binder(), V20201103145400_LegacyAuthServiceMigration.MigrationModule.class);
     }
 }
--- graylog2-server/src/main/java/org/graylog2/migrations/V20201103145400_LegacyAuthServiceMigration.java
+++ graylog2-server/src/main/java/org/graylog2/migrations/V20201103145400_LegacyAuthServiceMigration.java
@@ -58,6 +58,7 @@ import static org.apache.commons.lang3.StringUtils.isNotBlank;
 public class V20201103145400_LegacyAuthServiceMigration extends Migration {
     private static final Logger LOG = LoggerFactory.getLogger(V20201103145400_LegacyAuthServiceMigration.class);
 
+    // This migration is pluggable to allow us to modify the outcome from plugins (e.g. Enterprise)
     public interface MigrationModule {
         void upgrade(Document document, AuthServiceBackendDTO authServiceConfig);
     }
```